Seungju week2 task

seulnan opened the pull request:
전체적인 피드백 
1. 매우 높은 완성도...별점 오만개드립니다 고생많았어용
2. 파일구조는 assets 폴더에 보통 이미지&폰트를 분리해둡니다! 폴더도 두개로 분리해서 하나의 폴더에 하나의 과제가 들어가도록 파일구조를 바꿔봅시다!
3. css파일은 따로 분리하는게 좋아용

layout
1. 배경이미지 디자인대로 넣어주세용!
2. absolute속성을 이용했던데 멘토링에서도 말했듯이 반응형디자인할때 많이 깨집니다...flex를 이용해봅시다!
4. box-sizing: border-box;이용한거 진짜 별점오백만개;;;너무좋아요!!!

recipe
1. instructions에서 숫자 컬러 디자인대로 수정부탁드려용
2. nutrition에 여백값이 너무 작아용..다른친구들은 span을 많이썼던데 flex&div를 사용한건 아주 굿굿 
3. class이름을 안에 담긴 내용으로 해서 가시성있게 이름지은 점 칭찬해용 
5.  구분선 길이가 디자인보다 살짝 짧은것같은데 수정부탁드려용

dbstmdw committed:
[Feat] 나머지 기능 추가

Feat : 나머지 기능 추가

Simple Omelette Recipe 전체 구조 완성, Ingredients, Instruction, Nutrition 추가

dbstmdw committed:
[Feat] layout web 디자인

Feat : layout wed 디자인

layout 배경, 폰트, 박스 6개 구현 완료

Resolves : #1

dbstmdw committed:
[Fix] 텍스트와 불릿 사이 간격조정

Fix : 텍스트와 불릿 사이 간격조정

불릿, 숫자 등과 텍스트의 간격이 좁아서 px수정

Resolves : #2

dbstmdw committed:
Merge branch 'Seungju' of https://github.com/seulnan/HTML-to-React into Seungju

eunseo9311 commented on index.html:
https://github.com/media screen and (max-width: 768px) {
.desktop {
width: 100%;
height: auto;
padding: 20px;
}
.main-text, .sub-text {
width: auto;
font-size: smaller;
}
}

위 코드와 같이 반응형 디자인 적용하여 미디어 쿼리를 추가하여 모바일 환경에서도 잘 작동할 수 있도록 크기 및 레이아웃을 조정하는 것을 권장합니다🥰

dbstmdw committed:
[media-query 실습과제제출]

media-query 실습코드

dbstmdw committed:
[Fix]미디어쿼리 적용 및 코드리뷰반영

Fix : 미디어쿼리 적용 및 코드리뷰반영

flex를 이용한 코드로 수정, media query를 사용한 모바일 수준의 화면 조정, 배경 디자인대로 수정, css파일 구분.

dbstmdw committed:
[Fix] 레시피 코드리뷰반영

Fix : 레시피 코드리뷰 반영

instructions에서 숫자 컬러 디자인대로 수정, nutrition에 여백값 넓힘, 구분선길이 700px로 수정

dbstmdw committed:
[Feat] 아 망했당

dbstmdw committed:
[Fix] 레시피 최종본

Fix : 레시피 최종본

이제 레시피 파일 업로드 제대로 되었을까요...?

dbstmdw committed:
[Fix] 파일수정

Fix : 파일 수정

잘 올라가졌는지 확인부탁드려용,,

dbstmdw committed:
[Feat] 최종파일확인

Feat : 최종파일확인

최종 파일 확인 부탁,,,

dbstmdw committed:
[Fix] 미디어쿼리실습최종제출

Fix : 미디어쿼리실습최종제출

dbstmdw committed:
[Fix] 레이아웃 최종 제출

Fix : 레이아웃 최종 제출

dbstmdw committed:
[Fix] 레이아웃 수정

Fix : 레이아웃 수정

그룹박스 사이즈, 박스 사이즈, 폰트사이즈 수정

dbstmdw committed:
[Feat] todo 기본틀 구현

feat : todo 기본틀 구현

배경과 할일 목록 구현, hover와 filter들 구현, 체크표시와 라이트모드 다크모드 구현 했습니다.

dbstmdw committed:
[Feat] 구분선구현 및 필터스타일적용

Feat : 구분선구현 및 필터스타일적용

필터를 호버했을때 스타일이 바뀌게 하였고 클릭했을때 파란색으로 바뀌게 했습니다. task들 사이 구분선도 구현했습니다

dbstmdw committed:
[Feat] 드래그 앤 드롭 추가 및 필터수정

Feat : 드래그 앤 드롭 추가 및 필터 수정

드래그 앤 드롭 이벤트 추가했고, 필터 부분 디자인에 맞게 수정했습니다.

dbstmdw committed:
[Fix] 구분선 스타일 변경, 체크박스 호버 변경

Fix : 구분선 스타일 변경, 체크박스 호버 변경

구분선 다크버전일때 색 디자인에 맞게 변경했고, 길이 조정했습니다.
체크박스 호버했을때 파란 테두리 나타나게 했습니다.
리스트 호버할때 배경생기는 것도 없앴습니다.

dbstmdw committed:
[Fix] 다크모드 체크박스,그림자 색변경

Fix: 다크모드 체크박스,그림자 색변경

다크모드에서 체크박스와 그림자 색변경했습니다.

dbstmdw committed:
[Fix] 다크모드 체크박스,그림자 색변경

[Fix] 다크모드 체크박스,그림자 색변경

svg파일 추가 제출

dbstmdw committed:
[Fix] 체크박스 수정

Fix : 체크박스 수정

클릭 시 체크표시가 나오게 수정하였습니다.

dbstmdw committed:
[Fix] 체크박스 테두리 수정

Fix : 체크박스 테두리 수정

체크박스 호버했을때 테두리만 적용되게 수정했습니다.

dbstmdw committed:
[Feat] 리액트실습코드제출

Feat : 리액트실습코드제출

dbstmdw committed:
리액트 실습코드 제출

리액트 실습코드 제출

dbstmdw committed:
reactstudy 파일 삭제 및 제출

reactstudy 파일 삭제하고 제출합니다
